packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -20,57 +70,58 @@
         <li>left</li>
         <li class="default">right</li>
       </ol>
-      <p>If no
-        <span class="dummy-code">listPostion</span>
-        is defined, it will automatically be positioned to the right.</p>
+    </dd>
+    <dt>width <code>string</code></dt>
+    <dd>
+      <p>Acceptable values: any valid CSS width (px, rem, etc)</p>
+      <p><em>Notice: by default the dropdown list has a
+          <code class="dummy-code">min-width</code>
+          of
+          <code class="dummy-code">200px</code>
+          and a
+          <code class="dummy-code">max-width</code>
+          of
+          <code class="dummy-code">400px</code>
+          applied to it, so it adapts to the content size. If a
+          <code class="dummy-code">@width</code>
+          parameter is provided then the list will have a fixed width.</em></p>
     </dd>
     <dt>onClose <code>function</code></dt>
     <dd>
       <p>Callback function invoked when the dropdown is closed (if provided).</p>
     </dd>
+    <dt>“splattributes”</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
   </dl>
-  <hr />
-  <p class="dummy-paragraph" id="toggle-button-api">
-    This is the API for the
-    <span class="dummy-code">Dropdown::Toggle::Button</span>
-    component:
-  </p>
+
+  <h4 class="dummy-h4">Toggle::Button</h4>
+  <p class="dummy-paragraph">Here is the API for the "button-like" toggle component (yielded as
+    <code class="dummy-code">ToggleButton</code>):</p>
```

Comment:
```suggestion
  <p class="dummy-paragraph" id="toggle-button-api">Here is the API for the text toggle button component:</p>
```

[issue] missing ID whose value is used for the `<dl>` aria-labelledby` attribute.
[issue] edited wording for clarity and brevity.

Reply:
https://github.com/hashicorp/design-system/pull/235/commits/403789977cba3d2575a624b790df113b122014ea

Reply:
reworded with `yielded in a hash under the key`